Source/Language/Flow/SetupSequencePhrase.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace Moq.Language.Flow
+{
+	// keeping the fluent API separate from `SequenceMethodCall` saves us from having to
+	// define a generic variant `SequenceMethodCallReturn<TResult>`, which would be much more
+	// work that having a generic fluent API counterpart `SequenceMethodCall<TResult>`.
+	internal sealed class SetupSequencePhrase : ISetupSequentialAction
+	{
+		private SequenceMethodCall setup;
+
+		public SetupSequencePhrase(SequenceMethodCall setup)
+		{
+			this.setup = setup;
+		}
+
+		public ISetupSequentialAction Pass()
+		{
+			this.setup.AddPass();
+			return this;
+		}
+
+		public ISetupSequentialAction Throws<TException>()
+			where TException : Exception, new()
+			=> this.Throws(new TException());
```

Comment:
Perhaps delayed instantiation should be preferred over newing up exception objects way in advance?